app/src/main/java/net/kuama/documentscanner/viewmodels/CropperModel.kt

```diff
@@ -22,12 +23,13 @@ class CropperModel : ViewModel() {
     val originalBitmap = MutableLiveData<Bitmap>()
     val bitmapToCrop = MutableLiveData<Bitmap>()
 
-    fun onViewCreated(uri: Uri, contentResolver: ContentResolver) {
+    fun onViewCreated(uri: Uri, screenOrientationDeg: Int?, contentResolver: ContentResolver) {
```

Comment:
if you choose to have a default value
```suggestion
    fun onViewCreated(uri: Uri, screenOrientationDeg: Int, contentResolver: ContentResolver) {
```